### PR TITLE
net/netmgr: Fix typo

### PR DESCRIPTION
**Pull request by junmin-kim**
sock_type of UDS is TR_UDS

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c6308084e6bbf4192480dc8904113ac6f74096b2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c6308084e6bbf4192480dc8904113ac6f74096b2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9cf26c3c2e81d600e8d24b733e03dff63a7707c0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9cf26c3c2e81d600e8d24b733e03dff63a7707c0] - Code Rule Check OK.

File: os/net/net_initialize_netmgr.c [hunk 1]
```diff
@@ -26,7 +26,9 @@ struct tr_netmgr {
 	void *dev;
 };
 
+#ifdef CONFIG_LWNL80211
```
Comment (pillip8282) [inline os/net/net_initialize_netmgr.c]: I'll add dependency to Kconfig. so please remove this

File: os/net/netmgr/bsd_socket_api.c [hunk 2]
```diff
@@ -34,6 +34,10 @@
 #include "netstack.h"
 #include "uds.h"
 
+#ifndef CONFIG_NUDS_DESCRIPTORS
+#define CONFIG_NUDS_DESCRIPTORS 0
+#endif
```
Comment (pillip8282) [inline os/net/netmgr/bsd_socket_api.c]: how about to move it to uds.h?

File: os/net/net_initialize_netmgr.c (hunk 1 above)
Comment (junmin-kim) [reply os/net/net_initialize_netmgr.c]: Done.

File: os/net/netmgr/bsd_socket_api.c (hunk 2 above)
Comment (junmin-kim) [reply os/net/netmgr/bsd_socket_api.c]: Okay. I'll do it.

**Comment by seinfra** [comment]
Target : [ccc7308e89db4038e7a877aff5c2930eaaabf2d1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ccc7308e89db4038e7a877aff5c2930eaaabf2d1] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [81bdf126061033062c034887422464f7fcb6e230] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [81bdf126061033062c034887422464f7fcb6e230] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a933e6218d3e897c678df3ad69c9bfe1bcb09175] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a933e6218d3e897c678df3ad69c9bfe1bcb09175] - Code Rule Check OK.

**Pull request merged**